test/extensions/quic_listeners/quiche/envoy_quic_dispatcher_test.cc

```diff
@@ -237,6 +236,7 @@ INSTANTIATE_TEST_SUITE_P(EnvoyQuicDispatcherTests, EnvoyQuicDispatcherTest,
                          testing::ValuesIn(generateTestParam()), testParamsToString);
 
 TEST_P(EnvoyQuicDispatcherTest, CreateNewConnectionUponCHLO) {
+  quic::SetVerbosityLogThreshold(1);
```

Comment:
was this for test debug, or something we want on permanently?

Reply:
It was for debugging, removed now.